sdk/devcenter/azure-developer-devcenter/samples/create_devbox_sample.py

```diff
@@ -31,47 +31,64 @@
 from azure.identity import DefaultAzureCredential
 from azure.core.exceptions import HttpResponseError
 
+"""
+FILE: create_devbox_sample.py
+
+DESCRIPTION:
+    This sample demonstrates how to create, connect and delete a dev box using python DevCenterClient. For this sample,
+    you must have previously configured DevCenter, Project, Network Connection, Dev Box Definition, and Pool.More details 
+    on how to configure those requirements at https://learn.microsoft.com/azure/dev-box/quickstart-configure-dev-box-service
+
+
+USAGE:
+    python create_devbox_sample.py
+
+    Set the environment variables with your own values before running the sample:
+    1) DEVCENTER_ENDPOINT - the endpoint for your devcenter
+"""
+
 def get_project_name(LOG, client):
     projects = list(client.projects.list_by_dev_center(top=1))
     return projects[0].name
 
+
 def main():
-    logging.basicConfig(level=logging.DEBUG)
-    LOG = logging.getLogger()
 
     # Set the values of the dev center endpoint, client ID, and client secret of the AAD application as environment variables:
     # DEVCENTER_ENDPOINT, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET
     try:
         endpoint = os.environ["DEVCENTER_ENDPOINT"]
     except KeyError:
-        LOG.error("Missing environment variable 'DEVCENTER_ENDPOINT' - please set it before running the example")
+        print("Missing environment variable 'DEVCENTER_ENDPOINT' - please set it before running the example")
         exit()
```

Comment:
Nit, one more small fix here:
```suggestion
        raise ValueError("Missing environment variable 'DEVCENTER_ENDPOINT' - please set it before running the example")
```